lbry/wallet/server/db/elasticsearch/search.py

```diff
@@ -484,9 +486,8 @@ def expand_query(**kwargs):
             if not many and key in ('_id', 'claim_id') and len(value) < 20:
                 partial_id = True
             if key == 'public_key_id':
-                key = 'public_key_hash'
                 value = Base58.decode(value)[1:21].hex()
-            if key == 'signature_valid':
+            if key in ('signature_valid', 'has_source'):
```

Comment:
shoudln't this be `is_signature_valid`

Reply:
or is this after the transformation?

Reply:
`signature_valid` is part of the public API, so it comes in the request. It gets translated to `is_signature_valid` some lines later, when generating the final ES query from the API claim_search parameters